platform/android/export/export.cpp

```diff
@@ -2038,6 +2045,13 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 
 		String rk = p_preset->get("keystore/release");
+		String rk_user = p_preset->get("keystore/release_user");
+		String rk_password = p_preset->get("keystore/release_password");
+
+		if ((rk.is_empty() || rk_user.is_empty() || rk_password.is_empty()) && (!rk.is_empty() || !rk_user.is_empty() || !rk_password.is_empty())) {
+			valid = false;
+			err += TTR("Either Release keystore, Release User AND Release Password presets must be configured OR none of them.") + "\n";
```

Comment:
Use consistent capitalization and terminology:

```suggestion
			err += TTR("Either Release Keystore, Release User AND Release Password settings must be configured OR none of them.") + "\n";
```